tf-modules/archive/async_operation.tf

```diff
@@ -21,6 +21,10 @@ resource "aws_ecs_task_definition" "async_operation" {
       {
         "name": "databaseCredentialSecretArn",
         "value": "${var.rds_user_access_secret_arn}"
+      },
+      {
+        "name": "ES_HOST",
+        "value": "${var.elasticsearch_hostname}"
```

Comment:
Does this need to be noted in the changelog?

Reply:
I can for thoroughness, yeah

Reply:
fe15320a8aad50bceeb6ecaf4aed0eb143a7e7ff